db/schema.rb

```diff
@@ -35,6 +35,17 @@
     t.integer "department_id"
   end
 
+  create_table "courses_tutoring_sessions", id: false, force: :cascade do |t|
+    t.bigint "course_id", null: false
+    t.bigint "tutoring_session_id", null: false
+  end
+
+  create_table "courses_users", id: false, force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.bigint "course_id", null: false
+    t.string "grade_achieved"
+  end
+
```

Comment:
This and other additions are incorrect. The format is "singular"_"plural". I don't know why there are duplicates in here

Reply:
I'm not sure why these remained after renaming. Does renaming just make a copy? Regardless, I dropped the plural_plural tables